lib/internal/errors.js

```diff
@@ -38,14 +38,16 @@ function getMessage (key, args, self) {
     return ReflectApply(msg, self, args)
   }
 
-  const expectedLength =
-    (StringPrototypeMatch(msg, /%[dfijoOs]/g) || []).length
+  const expectedLength = (StringPrototypeMatch(msg, /%[dfijoOs]/g) || [])
+    .length
```

Comment:
about the style changes, shall we move away from `standard` to match node core closer (with semicolons and all)?

Reply:
Ugh, that's my VSCode config that's messing with me, trying to run Prettier on standard files

Reply:
I was using prettier-standard before but realized it was unmaintained